### PR TITLE
Add constants for IpBlockType types

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

**Review by jen20 — COMMENTED** [review]
Do we want to move these into the newer wrapper class style?

File: sdk/nodejs/billingCycle.ts [hunk 1]
```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const HourlyBilling:  BillingCycle = "hourly";
-export const MonthlyBilling: BillingCycle = "monthly";
```
Comment (jen20) [inline sdk/nodejs/billingCycle.ts]: If we made a release with these in we should probably keep them for a while and mark as `@deprecated`?

**Review by jen20 — APPROVED** [review]
LGTM pending question about breaking changes in releases.

File: sdk/nodejs/billingCycle.ts (hunk 1 above)
Comment (stack72) [reply sdk/nodejs/billingCycle.ts]: @jen20 none of these constants are in the public yet - just in master

**Pull request merged**